### DISCUSSION TITLE
Invoking the command line when in fullscreen mode

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1365] [Questions] [answered]
I was wondering if there is any way to invoke the command line when in fullscreen mode. When I try to do so by pressing `esc` Monkeytype exits fullscreen mode because `esc` is the default key set for that action in a browser.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
So you mean being able to open the command line without having to use escape? I figure @Miodec could make a button for the command line like he did for mobile users.

**Reply by Miodec** [reply]
Having a button where you must use your mouse kinda defeats the purpose. Im not sure whats a good way of going about this issue. I don't want to stop the default behaviour of Esc because then people won't be able to exit fullscreen (I'm not sure if the browser will even allow for this)

**Reply by unrealapex** [reply]
Possibly using a different key to invoke the command line like `/`?

**Reply by Miodec** [reply]
/ is used in tribe so can't do that

**Reply by unrealapex** [reply]
Could you reexplain what tribe is?

**Reply by Miodec** [reply]
Tribe is multiplayer. / focuses the chat box

**Reply by unrealapex** [reply]
Thanks for the clarification!

**Reply by Whydidthechickencrosstheroad** [reply]
Try like / key but disables when it is in game but when your side of the race is over, the / key enables it again to chat
